packages/webapp/src/elections/api/fixtures.ts

```diff
@@ -31,6 +31,12 @@ export const fixtureElectionState: ElectionState = {
     last_election_time: "2022-01-16T16:00:00.000",
 };
 
+export const fixtureRegistrationElection: CurrentElection = {
+    electionState: "registration?", // TODO: confirm?
+    start_time: "2022-01-16T16:00:00.000Z",
```

Comment:
So, the data from the API comes back _without_ the `Z` at the end. That means that when we're using dev fixtures, the UI will account for time zone properly (because you have the `Z` here) but when we fetch from the API, the UI will not properly adjust for time zone, as there's no `Z`.

I recommend we handle this by appending the `Z` manually (and in the same place whether fixtures or fetching). Maybe in `getCurrentElection`?

I'm open to suggestions.

Reply:
Nice Catch! Yeah I will replicate the fixture without Z and concatenate it manually!